### PR TITLE
Fix update old series view issue with no metrics ingested

### LINKED ISSUE #330: Unit-tests for fresh install/database
As in community channel, the 0.1.1-dev version has a `foreach null issue` on a fresh db. So, we need to have a dedicated test suite that works the installation of promscale schemas and normal functionality on a fresh database. This can be related to upgrades as well.

### PR DESCRIPTION
**Pull request by antekresic**
Fixes #330

This is a WIP mostly because the testing setup is too slow to run twice (once with data, once with no data ingested) and needs some work to get it running faster.

### CODE REVIEW

**Review by JLockerman — APPROVED** [review]
The SQL looks good.

CC @cevian on the fact that we're changing an upgrade script. Since it's a non-functional change I think it should be allowed

File: pkg/pgmodel/upgrade_tests/upgrade_test.go [hunk 1]
```diff
@@ -421,7 +444,7 @@ func toPreviousVersion(version *semver.Version) {
 	version.Pre = nil
 
 	if version.Patch > 0 {
-		version.Patch -= 1
+		version.Patch = 0
```
Comment (cevian) [inline pkg/pgmodel/upgrade_tests/upgrade_test.go:447]: This is not correct in general. But I think we are gonna change this logic anyway and this is needed for the PR so I think its ok

**Review by cevian — APPROVED** [review]
Looks good. I think we can optimize the testing later.

**Comment by cevian** [comment]
Note PR #336 is relevant here.

**Pull request merged**